sdk/tests/deqp/modules/shared/glsFboCompletenessTests.js

```diff
@@ -544,9 +544,6 @@ goog.scope(function() {
         sizeTests.addChild(new glsFboCompletenessTests.EmptyImageTest(
             'zero', 'Test for zero-sized image attachment', this
         ));
-        sizeTests.addChild(new glsFboCompletenessTests.DistinctSizeTest(
-            'distinct', 'Test for attachments with different sizes', this
-        ));
 
```

Comment:
I assume you have more code to delete than just this addChild? The DistinctSizeTes can all go?